### DISCUSSION TITLE
Water animation

### DISCUSSION THREAD

**Discussion by ZeljkoBG** [osmandapp/OsmAnd#20681] [Ideas]
Look at the water animation! Very hard! Very nice!
https://youtu.be/dlr33xf2e9w?t=220

Let's go, developers... add water, trees,... animation

**Comment by mariush444** [comment]
Do you have phone with to much performance power?

**Reply by ZeljkoBG** [reply]
I have the weakest phone and everything works flawlessly.
It's all about programming.

**Reply by sjvudp** [reply]
Maybe get your eyes off the phone and see where you are. It's as realistic as it can get!
Or do you want the sound of water, traffic sound, moving cars and pedestrians, rain and snow animation, sound of wind,...???
It's a navigation tool basically: Have a look at a compass: There are no animations at all, and still it can get you there (once you know how to use it).

**Reply by ZeljkoBG** [reply]
I agree that it is 'makeup'. But NNG knows best what to do with make-up. Is it redundant/unnecessary? Is.
But OsmAnd is on top with unnecessary 'extras' etc.
